doc-content/enterprise-only/openshift/template-overview-con.adoc

```diff
@@ -29,6 +29,12 @@ endif::[]
 ifeval::["{context}"=="openshift-trial"]
 * `rhpam72-trial-ephemeral.yaml` provides a {CENTRAL} and a {KIE_SERVER} connected to the {CENTRAL}. This environment uses an ephemeral configuration without any persistent storage. For details about this template, see <<rhpam72-trial-ephemeral-ref>>.
 endif::[]
+
+ifeval::["{context}"=="openshift-freeform"]
+* `rhpam73-managed.yaml` provides a high-availability {CENTRAL} Monitoring instance, a {KIE_SERVER}, and a PostgreSQL instance that the {KIE_SERVER} uses. OpenShiftStartupStrategy is enabled, ensuring that the {CENTRAL} Monitoring instance can connect to other {KIE_SERVER} instances in the same project automatically, as long as these instances have OpenShiftStartupStrategy enabled as well.
```

Comment:
I don't see this in the rendered draft but if you are using the file, then I would think that OpenShiftStartupStrategy should be code